lib/mayaUsd/fileio/utils/writeUtil.cpp

```diff
@@ -647,7 +649,9 @@ bool UsdMayaWriteUtil::SetUsdAttr(
         return true;
     }
 
-    VtValue val = GetVtValue(attrPlug, usdAttr.GetTypeName());
+    MDGContext      dgContext(MAnimControl::currentTime());
```

Comment:
Evaluating in the time context can be very expensive and in some cases incorrect. I'm not feeling comfortable changing this code.